app/helpers/samples_helper.rb

```diff
@@ -588,7 +588,7 @@ def self.samples_by_metadata_field(sample_ids, field_name)
 
   # For each taxon, count how many samples have taxon counts for that taxon.
   # Add these counts to the taxon objects.
-  def augment_taxon_list_with_sample_count(taxon_list, samples)
+  def augment_taxon_list_with_sample_count_reads(taxon_list, samples)
```

Comment:
This is still a sample count correct? Find it a bit confusing name. Seems like you counting the reads.
If so, do we really need to add `reads` in the end. I would either leave the name as it was or do `augment_taxon_list_with_sample_with_reads_count`.